src/pybind/mgr/dashboard_v2/frontend/src/app/shared/datatable/table/table.component.ts

```diff
@@ -131,19 +167,37 @@ export class TableComponent implements AfterContentChecked, OnInit, OnChanges {
   }
 
   reloadData() {
```

Comment:
When auto fetching it should also show the loading indicator.
That way the user knows that data is being fetched.

Reply:
See my other comment. It should not been shown as the indicator of the table moves the rows down a bit and this is really irritating if you have multiple rows. In the future we could rotated the reload icon, this would not distract the user that much.